Apps/AdminWebClient/test/unit/Services.Test/AuthenticationServiceTests.cs

```diff
@@ -131,16 +132,16 @@ public void ShouldUpdateLastLoginDateTimeWhenAdminUserProfileExists()
 
     private static IConfigurationRoot GetIConfigurationRoot()
     {
-        Dictionary<string, string> myConfiguration = new()
+        Dictionary<string, string?> myConfiguration = new()
         {
             { "EnabledRoles", "[ \"AdminUser\", \"AdminReviewer\", \"SupportUser\" ]" },
         };
 
         return new ConfigurationBuilder()
-            .AddJsonFile("appsettings.json", true)
-            .AddJsonFile("appsettings.Development.json", true)
-            .AddJsonFile("appsettings.local.json", true)
-            .AddInMemoryCollection(myConfiguration)
+            .AddJsonFile("appsettings.json", optional: true)
+            .AddJsonFile("appsettings.Development.json", optional: true)
+            .AddJsonFile("appsettings.local.json", optional: true)
```

Comment:
Our code cleanup profile "HG Full Cleanup" should be automatically removing named parameters, not adding them.  (Applies to `UserEmailServiceTests.cs` and `ImmunizationDelegateTests.cs` as well.)